src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-command.js

```diff
@@ -0,0 +1,13 @@
+import Command from '@ckeditor/ckeditor5-core/src/command';
+
+class IbexaCustomAttributesCommand extends Command {
+    refresh() {
+        const parentElement = this.editor.model.document.selection.getFirstPosition().parent;
+        const isEnabled =
+            window.eZ.richText.alloyEditor.attributes[parentElement.name] || window.eZ.richText.alloyEditor.classes[parentElement.name];
+
+        this.isEnabled = isEnabled;
```

Comment:
maybe `this.isEnabled = !!isEnabled` ? I'm not sure what line 7 returns, but I gues that's not bool

Reply:
Typescript maybe? :wink: 

:+1: It will definitely not be a boolean.

Reply:
> Typescript maybe? 😉

Nope